apps/eth-wrapper/README.md

```diff
@@ -0,0 +1,7 @@
+# Eth-Wrapper
+
+## Usage
+
+Simple safe app to wrap ETH or unwrap WETH.
```

Comment:
Could you add how to build the app and run tests. E.g. 
- What npm version (or an `.nvmrc` file, see:  https://til.hashrocket.com/posts/nkdafd69bi-create-an-nvm-config-file)
- What package manager (e.g. yarn or npm)
- What command to run tests/ lint or similar